LBH.AdultSocialCare.Api/V1/AppConstants/Enums/PackageStageEnum.cs

```diff
@@ -3,7 +3,7 @@
 namespace LBH.AdultSocialCare.Api.V1.AppConstants.Enums
 {
     // TODO: VK: There is a discrepancy between stages in constants and DB - what is actual?
-    public enum PackageStage
+    public enum PackageStageEnum
```

Comment:
In object-oriented design types represent objects and concepts of real world. In real world we have concept of Status (or Type, or Period or whatever). And we talk and think about them as Status, Type and Period, but not as Status Enum or Type Enum. Enum it or not - is just an implementation detail.

Thats's why Microsoft and most mature framework vendors don't use -Enum suffix in their frameworks (HttpStatus.Ok vs HttpStatusEnum.Ok).
That's why Microsoft explicitly denies usage of -Enum suffix in its naming guidelines
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations

Now we have a chance to get rid of bad habits - let's do it

Reply:
Fair point. Thinking behind this was I am going to also have a `PackageStatus` entity as well and I am trying to have different names to avoid that confusion. Guess I have to find a different name now